src/pudl/output/pudltabl.py

```diff
@@ -82,15 +82,14 @@ def __init__(self, pudl_engine, ds=None, freq=None, start_date=None, end_date=No
         if start_date is None:
             self.start_date = \
                 pd.to_datetime(
-                    f"{min(pc.working_years['eia923'])}-01-01")
+                    f"{min(pc.working_partitions['eia923']['years'])}-01-01")
```

Comment:
I'm wondering if this conversion could be done better.

One idea here would be to use `pandas.Timestamp` object which can be converted to datetime with `as_datetime64` method and that can be instantiated nicely. In this case:

```
self.start_date = pandas.Timestamp(year=min(pc.working_partitions['eia923']['years']))
```